sycl/test/regression/ambigious-memcpy.cpp

```diff
@@ -0,0 +1,17 @@
+// detail::memcpy, even though in a different namespace, can cause ambiguity
+// with libc's memcpy, due to argument dependent lookup (ADL). This compile-only
+// test checks for ambigiouity in call to detail::memcpy.
+
+// RUN: %clang -fsycl -fsycl-device-only -fsyntax-only %s
```

Comment:
Tiny nit and by no means blocking, but normally we have the REQUIRES and RUN lines first and the description after.

```suggestion
// RUN: %clang -fsycl -fsycl-device-only -fsyntax-only %s

// detail::memcpy, even though in a different namespace, can cause ambiguity
// with libc's memcpy, due to argument dependent lookup (ADL). This compile-only
// test checks for ambiguity in call to detail::memcpy.
```

Note also a small spelling mistake fix in there: "ambigiouity" -> "ambiguity".